src/test/java/cloud/eppo/BaseEppoClientTest.java

```diff
@@ -44,185 +35,67 @@ public class BaseEppoClientTest {
   private static final String DUMMY_FLAG_API_KEY = "dummy-flags-api-key"; // Will load flags-v1
   private static final String TEST_HOST =
       "https://us-central1-eppo-qa.cloudfunctions.net/serveGitHubRacTestFile";
-  private final ObjectMapper mapper = new ObjectMapper().registerModule(module());
+  private final ObjectMapper mapper =
+      new ObjectMapper().registerModule(AssignmentTestCase.assignmentTestCaseModule());
 
+  private BaseEppoClient eppoClient;
```

Comment:
This is _not_ static because for these tests we often initialize a new client, so we use the two methods below to get a new client for each test as needed